docs/debuginfod.md

```diff
@@ -0,0 +1,37 @@
+# debuginfod client
+
+In order to symbolize ingested profiles, Parca needs to have debug information
+for the binaries _(which binaries?)_ that are being profiled. Debug information
```

Comment:
I don't understand what the "_(which binaries?)_" is intended for here. Are you asking?

Reply:
Sort of, just _binaries_ seemed incomplete to me (and would have confused me a few years ago maybe). I think I will go with application binaries. wdyt?

Reply:
I think application binaries is misleading because it doesn't matter what binary it is, any executed binary is automatically profiled.